Solutions/Commvault Security IQ/Playbooks/CommvaultLogicApp/readme.md

```diff
@@ -0,0 +1,20 @@
+# Commvault Logic App Playbook
+## Summary
+This Logic App executes when called upon by an Automation Rule. Accessing the KeyVault to retrieve various credentials, it executes a specific runbook depending on the use case. 
+
+## Prerequisites
+- Administrative access to your Commvault/Metallic environment.
+- Administrative access to your Azure Resource Group and Subscription.
+- An Azure Sentinel instance in the aforementioned Azure Resource Group.
+- A Keyvault and an Automation Account configured as mentioned in the documentation here :- (https://github.com/Cv-securityIQ/Azure-Integration/blob/Commvault/Solutions/Commvault%20Security%20IQ/README.md)
+
+## Deployment Instructions
```

Comment:
Readme file must have "Deploy to Azure" and "Deploy to Azure Gov".
(part of Deployment instructions)

Please Refer :- https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/ThreatXCloud/Playbooks/ThreatXPlaybooks/ThreatX-Enrichment/readme.md

Reply:
Hi @v-rbajaj, I've made all the suggested changes.
I've kept the "prerequisitesDeployTemplateFile":  "", and "entities": [] as empty since we are not using any data connectors and also the arm-ttk validation is still failing (Summary: @ {Pass=46; Fail=2; Total=48}). I've pushed the latest changes, please let me know what else needs to be done here.

Reply:
Hi @v-rbajaj, did you get a chance to check the failures?